src/factories/nodes.ts

```diff
@@ -82,6 +83,15 @@ export async function nodesContainerFactory() {
   }
 
   async function createEdges(data: RunGraphData): Promise<void> {
+    const settings = await waitForSettings()
+
+    if (settings.disableEdges) {
+      container.removeChild(edgesContainer)
+      return
+    }
+
+    container.addChildAt(edgesContainer, 0)
```

Comment:
`adChildAt` means edges will always be behind the rest of the node content.

Reply:
Heads up, in V1, a transparent rectangle is added to the viewport and also set behind nodes as a deselect target when a node is selected. Might be worth tracking these top level z-indexes (can do later of course)

Reply:
I'm hoping not to use sortable children since sorting large arrays is very slow. As long as we add them in the correct order we shouldn't need to use z-indexes. 

Also I'm using the viewport itself as the deselect target and using stopPropogation on the nodes themselves.